systemd/osdf-origin.yaml

```diff
@@ -0,0 +1,24 @@
+Debug: false
+Logging:
+  Level: "Error"
+Federation:
+  DiscoveryUrl: osg-htc.org
+  TopologyNamespaceURL: https://topology.opensciencegrid.org/osdf/namespaces
+Server:
+  TLSCertificate: /etc/grid-security/xrd/xrdcert.pem
+  TLSKey: /etc/grid-security/xrd/xrdkey.pem
+  TLSCACertificateDirectory: /etc/grid-security/certificates
+  EnableUI: false
+#  Hostname: f.q.d.n
+Xrootd:
+  Sitename: TOPOLOGY_RESOURCE_NAME
+  ManagerHost: redirector.osgstorage.org
+  SummaryMonitoringHost: xrd-report.osgstorage.org
+  DetailedMonitoringHost: xrd-mon.osgstorage.org
+  Mount: "/mnt/osdf"
+  Port: 1095
+#  Authfile: /run/stash-origin-auth/Authfile
```

Comment:
Let's remove commented-out lines (or add a comment about what they're for).

Reply:
The plan is to do the latter.

Reply:
Actually for now I'm going to uncomment them since we're going to need them until #564 is implemented.